File: code-coaches/medium/symbols/symbols.rb
def remove_symbols(encoded)
  # symbols: a char that is not alphanumeric or a space
  encoded.gsub(/[^[:alnum:][:space:]]/, '')
end

def main
  encoded = gets.chomp
  puts remove_symbols(encoded)
end

if __FILE__ == $0
  main
end

File: code-coaches/medium/symbols/Symbols.swift
func removeSymbols(from encoded: String) -> String {
    // symbols: a char that is not alphanumeric or a space
    return encoded.filter {
        let isLetter = ($0 >= "a" && $0 <= "z") || ($0 >= "A" && $0 <= "Z")
        let isNumber = $0 >= "0" && $0 <= "9"
        let isWhitespace = $0 == " "
        return isLetter || isNumber || isWhitespace
    }
}

func main() {
    let encoded = readLine()!
	print(removeSymbols(from: encoded))
}

main()